server/bin/pbench-index.py

```diff
@@ -153,16 +154,16 @@ def main(options, name):
 
     res = 0
     try:
-        ARCHIVE_rp = os.path.realpath(idxctx.config.ARCHIVE)
-        if not os.path.isdir(ARCHIVE_rp):
+        ARCHIVE_rp = Path(idxctx.config.ARCHIVE).resolve()
```

Comment:
I think we want to use `strict=True` option here.  We might want to make a helper method which takes a configuration value, turns it into a Path object, resolves it, and validates it is a directory, since we do that often.

Reply:
> It might be possible to abstract the repetitive checks for each directory into a utility function that can be imported and used by every program here (although there are some special cases: print vs logger.error and some checking is file checking, rather than directory checking).

Then should we consider the idea given by Nick?

Reply:
What is the idea given by Nick?